### ISSUE TITLE
Confidential info from database is printed in stdout

### ISSUE THREAD

**Issue by agilob** [qTox/qTox#3399]
##### Brief Description

OS: PC-BSD
qTox version: 1.4 (installed with `pkg` not from sources this time)
Commit hash:  NA
toxcore:  0.0.0
Qt:  5.5.1
Hardware:  
…

Reproducible: Always
##### Steps to reproduce
1. I just turn on qTox from terminal
##### Observed Behavior

I get this log on each start of qtox for each of my contacts:

```
[20:11:15.865] src/persistence/db/rawdatabase.cpp:374 : Warning: Failed to prepare statement 
"SELECT history.id, faux_offline_pending.id, timestamp, chat.public_key, aliases.display_name, 
sender.public_key, message FROM history LEFT JOIN faux_offline_pending ON history.id = 
faux_offline_pending.id JOIN peers chat ON chat_id = chat.id JOIN aliases ON sender_alias = 
aliases.id JOIN peers sender ON aliases.owner = sender.id WHERE timestamp BETWEEN 
146541300000 AND 14660000000 AND 
chat.public_key='PUBLIC KEY WAS HERE';" with error 26

...
a bit later:
...
[20:13:07.881] src/persistence/db/rawdatabase.cpp:374 : Warning: Failed to prepare statement "INSERT INTO peers (id, public_key) VALUES (1, 'PUBLIC KEY WAS HERE');" with error 26

```
##### Expected Behavior

public_key and time stamps should not be exposed in log.

**Comment by crevete** [comment]
I've also got this error message every time I launch qTox... And it fails to load chat history...
Can you tell me how to fix this please? I hope I didn't lose all my chat history.. Thanks

**Comment by ezavod** [comment]
@crevete These errors seem to be part of issue #4344. It should be fixed by PR #4347.
